### ISSUE TITLE
Job [Library sync for provider Filesystem] failed with error remove_prov_mapping() missing 1 required positional argument: 'prov_id'.

### ISSUE THREAD

**Issue by RJ-Make** [music-assistant/hass-music-assistant#451]
Just updated to 2022.6.2 and now I am seeing this error in the logs. 

```
Logger: music_assistant
Source: /usr/local/lib/python3.9/site-packages/music_assistant/mass.py:244
First occurred: 6:07:47 AM (1 occurrences)
Last logged: 6:07:47 AM

Job [Library sync for provider Filesystem] failed with error remove_prov_mapping() missing 1 required positional argument: 'prov_id'.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/music_providers/filesystem.py", line 251, in sync_library
    await self.mass.music.albums.remove_prov_mapping(album_id)
TypeError: remove_prov_mapping() missing 1 required positional argument: 'prov_id'
```

**Comment by marcelveldt** [comment]
Thanks for reporting. I'll provide a fix!

**Comment by marcelveldt** [comment]
fixed in 2022.6.3, please reopen if you still experience any issues, thanks!
